### ISSUE TITLE
JSON format to copy types

### ISSUE THREAD

**Issue by chlbri** [mxsdev/ts-type-explorer#53]
It will be nicer to have the possibility to copy the type inside code (json or directly in clipboard)

**Comment by max-pod** [comment]
Agreed. Stumbled on to this plugin looking for the same functionality

**Comment by soanvig** [comment]
I have the same - niche -  case.

I have very very complex computed type, that I want to "cache" by making it fixed in place - the easiest solution would be to grab copy the full type, do some manual adjustments, and save as type alias. However this extension doesn't provide such functionality, although I must admit that the explorer by itself is very nice.
